images/k8s-infra/cloudbuild.yaml

```diff
@@ -11,7 +11,6 @@ steps:
     - TAG=$_TAG
     - TAG_DATE_SHA=$_GIT_TAG
     - _=$_PULL_BASE_REF # unused
-serviceAccount: 'projects/$PROJECT_ID/serviceAccounts/gcb-image-builder@$PROJECT_ID.iam.gserviceaccount.com'
```

Comment:
? unrelated ?

Reply:
Added in https://github.com/kubernetes/k8s.io/pull/4028 to test GCB with dedicated service accounts. Not impactful for the PR and the jobs using this image.